src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -113,13 +113,17 @@ internal static int Start(string bannerText, string helpText, string[] args)
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-linux`
Pwsh startup and PATH.pwsh starts even if PATH is not defined
```powershell
Expected strings to be the same, but they were different.
Expected length: 33
Actual length:   314
Strings differ at index 33.
Expected: '/home/vsts/work/1/a/bins/publish:'
But was:  '...lish:/home...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 1009
1009:         $path | Should -BeExactly ($PSHOME + [System.IO.Path]::PathSeparator)
```